pkg/util/provider/machinecontroller/machine.go

```diff
@@ -134,6 +134,8 @@ func (c *controller) reconcileClusterMachine(machine *v1alpha1.Machine) (machine
 		return machineutils.LongRetry, err
 	}
 
+	// Validate MachineClass
+	klog.Infof("Validating Machine Class %s backing machine %s", machine.Spec.Class.Name, machine.Name)
```

Comment:
Do we need this?

Reply:
Yeah, it does not really make a difference. I will remove it